nebula_ros/src/hesai/hw_interface_wrapper.cpp

```diff
@@ -28,6 +30,11 @@ HesaiHwInterfaceWrapper::HesaiHwInterfaceWrapper(
   hw_interface_->SetLogger(std::make_shared<rclcpp::Logger>(parent_node->get_logger()));
   hw_interface_->SetTargetModel(config->sensor_model);
 
+  if (!communicate_with_sensor_) {
+    // Do not initialize Tcp if communication is disabled
+    return;
```

Comment:
`status_` is not updated before returning. Should be `Status::OK`.

Reply:
The `status_` should be set after sensor configuration is initialized ([here](https://github.com/msz-rai/nebula/blob/feature/disable-communication-option/nebula_ros/src/hesai/hw_interface_wrapper.cpp#L21)).

Reply:
Ah, indeed. My bad!